content/en/docs/contribute/review/for-approvers.md

```diff
@@ -0,0 +1,233 @@
+
+  {{< /table >}}
+
+  At your discretion, take ownership of an issue and submit a PR for it
+  (especially if it's quick or relates to work you're already doing).
+
+If you have questions about triaging an issue, ask in `#sig-docs` on Slack or
+the [kubernetes-sig-docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
+
+## Adding and removing issue labels
+
+To add a label, leave a comment in one of the following formats:
+
+- `/<label-to-add>` (for example, `/good-first-issue`)
+- `/<label-category> <label-to-add>` (for example, `/triage needs-information` or `/language ja`)
+
+To remove a label, leave a comment in one of the following formats:
+
+-`/remove-<label-to-remove>` (for example, `/remove-help`)
+- `/remove-<label-category> <label-to-remove>` (for example, `/remove-triage needs-information`)`
+
+In both cases, the label must already exist. If you try to add a label that does not exist, the command is
+silently ignored.
+
+For a list of all labels, see the [website repository's Labels section](https://github.com/kubernetes/website/labels). Not all labels are used by SIG Docs.
+
+### Issue lifecycle labels
+
+Issues are generally opened and closed quickly.
+However, sometimes an issue is inactive after its opened.
+Other times, an issue may need to remain open for longer than 90 days.
+
+{{< table caption="Issue labels" >}}
+Label | Description
+:------------|:------------------
+`lifecycle/stale` | After 90 days with no activity, an issue is automatically labeled as stale. The issue will be automatically closed if the lifecycle is not manually reverted using the `/remove-lifecycle stale` command.
+`lifecycle/frozen` | An issue with this label will not become stale after 90 days of inactivity. A user manually adds this label to issues that need to remain open for much longer than 90 days, such as those with a `priority/important-longterm` label.
+{{< /table >}}
+
+## Handling special issue types
+
+SIG Docs encounters the following types of issues often enough to document how
+to handle them.
+
+### Duplicate issues
+
+If a single problem has one or more issues open for it, combine them into a single issue.
+You should decide which issue to keep open (or
+open a new issue), then move over all relevant information and link related issues.
+Finally, label all other issues that describe the same problem with
+`triage/duplicate` and close them. Only having a single issue to work on reduces confusion
+and avoids duplicate work on the same problem.
+
+### Dead link issues
+
+If the dead link issue is in the API or `kubectl` documentation, assign them `/priority critical-urgent` until the problem is fully understood. Assign all other dead link issues `/priority important-longterm`, as they must be manually fixed.
```

Comment:
nit: Do you mean, "don't fix links in the [kubectl or API] reference documentation"?